packages/twenty-front/src/hooks/__tests__/useNavigateSettings.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { renderHook } from '@testing-library/react';
```

Comment:
I'm not 100% sure but I believe that we don't need these tests to be .tsx files .ts should be enough

Reply:
Yes thought the same but I didn't try changing to avoid getting even more stuff, I'll try in a followup PR!